### ISSUE TITLE
Mirage and adapter errors on Ember Server

### ISSUE THREAD

**Issue by amenders** [Renascentinc/retina-app#19]
On starting the Ember Server, the following errors are produced in the terminal, right before a `build successful` message.

```
mirage/config.js: line 52, col 21, 'params' is not defined.

1 error

===== 1 JSHint Error


adapters/application.js: line 2, col 8, 'DataAdapterMixin' is defined but never used.

1 error

===== 1 JSHint Error

```

**Comment by Jbcampbe** [comment]
these two should be easy fixes

**Comment by Jbcampbe** [comment]
Fixed with the merge of #35
